components/network/samba/Makefile

```diff
@@ -28,12 +28,13 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME =		samba
 COMPONENT_FMRI =		service/network/samba
-COMPONENT_VERSION =		4.10.6
-COMPONENT_SRC =			$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_VERSION =		4.11.0
+COMPONENT_SRC =			$(COMPONENT_NAME)-$(COMPONENT_VERSION)rc4
 COMPONENT_PROJECT_URL =		http://www.samba.org/
-COMPONENT_ARCHIVE =		$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH = 	sha256:9efbeb52db1203dc779b118f1c48c161e569f7a6af5101e745497ee6296eef42
-COMPONENT_ARCHIVE_URL =		http://us1.samba.org/samba/ftp/stable/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE =             $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH = 	sha256:29663121eb4ed259bb3bd6c3038b27255f4414a1cca120acd7f29493c22df37f
+COMPONENT_ARCHIVE_URL =		https://download.samba.org/pub/samba/rc/$(COMPONENT_ARCHIVE).tar.gz
```

Comment:
```suggestion
COMPONENT_ARCHIVE_URL =		https://download.samba.org/pub/samba/rc/$(COMPONENT_ARCHIVE)
```

Reply:
I know, but until the release is out I will keep it this way. Then I switch to the lines which are there from the beginning...